### ISSUE TITLE
Disable line wrapping in single select

### ISSUE THREAD

**Issue by phil294** [vscode-elements/elements#231]
![image](https://github.com/user-attachments/assets/8dca0c17-8482-4ff0-958f-99d171c44f69)

I'd like the select to always be single-line, like a normal `<select>`, but can't find option for that. Am I missing something or is this just not supported right now?

Here's what I am doing to work around it right now:

```js
let style = document.createElement('style')
style.innerHTML = 'text { display: block; overflow: hidden; height: 18px; }'
the_select.shadowRoot.appendChild(style)
```

**Comment by bendera** [comment]
It's not supported, as I haven't faced this issue before. :) Am I seeing this correctly? Did you place it in a flexbox?

**Comment by phil294** [comment]
Nope, not in a flex, but the container has a limited width. This problem basically always applies when the container is smaller than what the select wants to render, independent of the wrapping display property. And since my select contains user-provided strings, I have no control over the contents.

**Comment by bendera** [comment]
It's weird. I'm trying to reproduce the state you depicted above but I can't.

Long text without space:
![image](https://github.com/user-attachments/assets/fd5700a5-f231-4050-8a91-76bdc4053272)

Text is wrapped in the dropdown but not in the select face:
![image](https://github.com/user-attachments/assets/95135d1d-76a8-447a-9610-c8bd358bdbaa)

Which version do you use?
